src/components/CurrentUser/CurrentUser.tsx

```diff
@@ -1,12 +1,48 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import './CurrentUser.scss';
+import { User } from '../../types/type';
+import { getUserById } from '../../types/api';
 
-export const CurrentUser: React.FC = () => (
-  <div className="CurrentUser">
-    <h2 className="CurrentUser__title"><span>Selected user: 2</span></h2>
+type Props = {
+  selectedUserId: number,
+  getClear: () => void,
+};
 
-    <h3 className="CurrentUser__name">Ervin Howell</h3>
-    <p className="CurrentUser__email">Shanna@melissa.tv</p>
-    <p className="CurrentUser__phone">010-692-6593 x09125</p>
-  </div>
-);
+export const CurrentUser: React.FC<Props> = ({
+  selectedUserId,
+  getClear,
+}) => {
+  const [selectedUser, setSelectedUser] = useState<User>();
+
+  useEffect(() => {
+    getUserById(selectedUserId)
+      .then(data => setSelectedUser(data));
+  }, [selectedUserId]);
+
+  return (
+    <div className="CurrentUser">
+      {(!selectedUser)
+        ? 'Not found user'
+        : (
```

Comment:
let's change the order

```suggestion
      {selectedUser
        ? main markup
        : 'User not found'
```